azurerm/helpers/azure/batch_pool.go

```diff
@@ -518,3 +518,34 @@ func ValidateAzureRMBatchPoolName(v interface{}, k string) (warnings []string, e
 
 	return warnings, errors
 }
+
+// ExpandMetaData expands Batch pool metadata
+func ExpandMetaData(input map[string]interface{}) *[]batch.MetadataItem {
```

Comment:
since this package contains helpers for all different libraries - could we include the service name in the method:

```suggestion
func ExpandBatchMetaData(input map[string]interface{}) *[]batch.MetadataItem {
```

Reply:
done